packages/client/package.json

```diff
@@ -1,72 +1,74 @@
 {
   "name": "client",
-  "version": "0.1.0",
+  "version": "1.0.0",
```

Comment:
Is there a way we can pull this version from the main package.json so we don't have to update it in three places when we change it?

Reply:
Not really, I was able to remove it from root but version is not inherited by the downstream workspaces as they are designed to be versioned independently.

Reply:
https://classic.yarnpkg.com/en/docs/workspaces/